servers/text_server.h

```diff
@@ -414,8 +414,8 @@ class TextServerManager : public Object {
 		void *user_data = nullptr;
 	};
 
-	static TextServerCreate server_create_functions[MAX_SERVERS];
-	static int server_create_count;
+	static TextServerCreate server_create_functions[MAX_SERVERS] = {};
+	static int server_create_count = 0;
```

Comment:
These are static variables, this won't build. Static vars are initialized in the `.cpp`.

Reply:
Yeah, I am working on fixing them right now. Sorry.